web/spacewalk-web.changes.serpico.Manager-5.0-MU-5.0.4-password-policy-fix-2

```diff
@@ -0,0 +1 @@
+- Fix: better react state handling (bcs#1241140)
```

Comment:
I think this changelog could use a better wording. These entries are used for the changelog for the users and react state handling will probably not tell them much useful.